src/harness/sas_testcase.py

```diff
@@ -94,14 +94,10 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
       self.assertEqual(grant_resp['cbsdId'], cbsd_id)
       self.assertTrue(grant_resp['grantId'])
       self.assertEqual(grant_resp['response']['responseCode'], 0)
-
-      list_of_cbsd_ids_and_grant_ids.append({
-        'cbsdId': grant_resp['cbsdId'],
-        'grantId': grant_resp['grantId']
-      })
+      grant_ids.append(grant_resp['grantId'])
 
     # Return cbsd_ids and grant_ids
```

Comment:
Would you mind adding a quick "assertEqual(len(cbsd_ids), len(grant_ids))" just to be absolutely sure there's no discrepancy?